### ISSUE TITLE
Suport for IRC

### ISSUE THREAD

**Issue by pi3ch** [infosec-au/websec-weekly#6]
It would be great to push weekly news to an IRC channel [enhancement].

**Comment by infosec-au** [comment]
Good idea, I don't use IRC much myself, but this would be super easy to do since /managers/hipchat.py basically has the same concept... Might work on this when I start using IRC more often (soon).

**Comment by cyphar** [comment]
> not using IRC
